packages/php/email-editor/src/Integrations/Core/Renderer/Blocks/class-embed.php

```
@@ -0,0 +1,232 @@
+	}
+
+	/**
+	 * Render a simple link fallback for non-audio embeds.
+	 *
+	 * @param array             $attr Block attributes.
+	 * @param string            $block_content Block content.
+	 * @param array             $parsed_block Parsed block.
+	 * @param Rendering_Context $rendering_context Rendering context.
+	 * @return string Rendered link or empty string if no valid URL.
+	 */
+	private function render_link_fallback( array $attr, string $block_content, array $parsed_block, Rendering_Context $rendering_context ): string {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$rendering_context'. (undefined)

(UnusedFormalParameter)